flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/sink/shuffle/TestDataStatisticsOperator.java

```diff
@@ -118,20 +126,16 @@ public void testProcessElement() throws Exception {
         testHarness = createHarness(this.operator)) {
       StateInitializationContext stateContext = getStateContext();
       operator.initializeState(stateContext);
-      operator.processElement(new StreamRecord<>(GenericRowData.of(StringData.fromString("a"))));
-      operator.processElement(new StreamRecord<>(GenericRowData.of(StringData.fromString("a"))));
-      operator.processElement(new StreamRecord<>(GenericRowData.of(StringData.fromString("b"))));
+      operator.processElement(new StreamRecord<>(binaryRowDataA));
```

Comment:
I feel we don't have change these. I understood the Flink RowData deserialization creates `BinaryRowData` and we have to use them for equality comparison during assertions